### ISSUE TITLE
wasm2wast crashes when using -v flag

### ISSUE THREAD

**Issue by dschuff** [WebAssembly/wabt#181]
run wasm2wast with -v on any (AFACS) binary, and it will crash:

```
$ out/clang/Debug/wast2wasm f.wast -o f.wasm
$ out/clang/Debug/wasm2wast f.wasm 
(module
  (memory (;0;) 17)
  (export "memory" (memory 0)))
$ out/clang/Debug/wasm2wast f.wasm -v
begin_module
begin_memory_section
  on_memory_count(1)
  on_memory(index: 0, initial: 17)
end_memory_section
begin_export_section
  on_export_count(1)
  on_export(index: 0, kind: memory, item_index: 0, name: "memory")
end_export_section
Segmentation fault (core dumped)
```
Haven't investigated yet beyond that.

**Comment by binji** [comment]
Oh, I think I saw this and fixed it on a branch somewhere...

**Comment by binji** [comment]
Yeah, this is fixed in 5baecee28e617f351fb9bea788c522724bc5fd75. This was the fix:
```
--- a/src/wasm-binary-reader.c
+++ b/src/wasm-binary-reader.c
@@ -2015,7 +2015,7 @@ WasmResult wasm_read_binary(WasmAllocator* allocator,
     CALLBACK_CTX0(end_data_section);
   }
 
-  CALLBACK_CTX0(end_module);
+  CALLBACK0(end_module);
   destroy_context(allocator, ctx);
   return WASM_OK;
 }
```
